### ISSUE TITLE
ERROR: No matching distribution found for pandas>=1.0 (from mintapi)

### ISSUE THREAD

**Issue by ccaylward** [mrooney/mintapi#224]
Please help! when I enter:

`pip install mintapi`

I get this error message:

ERROR: Could not find a version that satisfies the requirement pandas>=1.0 (from mintapi) (from versions: 0.1, 0.2b0, 0.2b1, 0.2, 0.3.0b0, 0.3.0b2, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.5.0, 0.6.0, 0.6.1, 0.7.0rc1, 0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.8.0rc1, 0.8.0rc2, 0.8.0, 0.8.1, 0.9.0, 0.9.1, 0.10.0, 0.10.1, 0.11.0, 0.12.0, 0.13.0, 0.13.1, 0.14.0, 0.14.1, 0.15.0, 0.15.1, 0.15.2, 0.16.0, 0.16.1, 0.16.2, 0.17.0, 0.17.1, 0.18.0, 0.18.1, 0.19.0rc1, 0.19.0, 0.19.1, 0.19.2, 0.20.0rc1, 0.20.0, 0.20.1, 0.20.2, 0.20.3, 0.21.0rc1, 0.21.0, 0.21.1, 0.22.0, 0.23.0rc2, 0.23.0, 0.23.1, 0.23.2, 0.23.3, 0.23.4, 0.24.0rc1, 0.24.0, 0.24.1, 0.24.2)
ERROR: No matching distribution found for pandas>=1.0 (from mintapi)

I'm new to Python, and looking for answer to troubleshoot. 

Thanks!

**Comment by ccaylward** [comment]
Also, I'm on a mac, if that explains anything.

**Comment by mrooney** [comment]
Hey there, I can't reproduce this on a fresh Python 3.8 // pip 20.2.2
virtualenv on OSX. It grabs pandas 1.1.1: https://pypi.org/project/pandas/.
I'm not sure why your pip couldn't find it. What is your python and pip
version? (`python -V`, `pip --version`)?

**Comment by ccaylward** [comment]
Thanks for the response! I was able to troubleshoot. Looks like my mac terminal defaulted to python 2, despite having downloaded and installed python 3 on my computer. I was able to work around by entering: 

`pip3 install mintapi`

and then adding "python3" before the code in my terminal. That did the trick. Go figure. Hope this helps a future mac user in the future!

Now I just need to figure out how to use this code to pull data into Tableau... I would love suggestions if the community has any.
